in_toto/in_toto_mock.py

```diff
@@ -100,7 +80,15 @@ def main():
   # TRY out in-toto-run, with fewer command lines args and max. user feedback.
```

Comment:
I don't understand why this comment is here.  Should it go in the module docstring?

Reply:
Santiago requested to clarify the comment I had before an the PR that modified this line:
https://github.com/in-toto/in-toto/pull/183#discussion_r166432991.

The *"max. user feedback"* is the crucial part, that should explain why I set the log level to `INFO`.

Reply:
Oh, it wasn't obvious to me how the comment relates to `log.setLevel(...)`.  Consider explicitly stating the connection, as you proposed in #183.

"in-toto-mock should not be used to secure the supply chain but only to try out in-toto-run, **hence it gives verbose user feedback**"